grakn-dashboard/src/js/HAL/HALParser.js

```diff
@@ -20,6 +20,8 @@
 
 import * as API from '../util/HALTerms';
 import * as Utils from './APIUtils';
+import _ from 'underscore';
```

Comment:
JS is a strange place

Reply:
I used to use this library back in the day. Agreed.